client/src/ui/components/friends/ChatInputBox.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane } from "@fortawesome/free-solid-svg-icons";
+
+interface ChatInputBoxProps {
+  setTextToSend: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const ChatInputBox: React.FC<ChatInputBoxProps> = ({ setTextToSend }) => {
```

Comment:
rather than using `React.FC`, could you declare the prop types in the function arguments [like you do here?](https://github.com/varangill/Synker/pull/13/files#diff-aa39d3c3bf6877b8c95c9584154608bcaa48682e04416750c6c9f39d83cf5714R3) i think it's cleaner

i.e. `const ChatInputBox = (setTextToSend: correcttype) => ...`

same with other new components and in the future